dsp/window/window_test.go

```diff
@@ -168,20 +195,21 @@ func TestWindows(t *testing.T) {
 	}
 }
 
-func TestGausWindows(t *testing.T) {
+func TestMonoParametricWindows(t *testing.T) {
 	t.Parallel()
 	const tol = 1e-6
 
-	for _, test := range gausWindowTests {
-		t.Run(fmt.Sprintf("%s (sigma=%.1f)", test.name, test.sigma), func(t *testing.T) {
+	for _, test := range monoParamWindowTests {
+		t.Run(fmt.Sprintf("%s (param=%.1f)", test.name, test.param), func(t *testing.T) {
 			src := make([]float64, 20)
 			for i := range src {
 				src[i] = 1
 			}
-
-			gaussian := Gaussian{test.sigma}
-
-			dst := gaussian.Transform(src)
+			type transformer interface {
+				Transform([]float64) []float64
+			}
+			trans := []transformer{Gaussian{test.param}, Tukey{test.param}}[test.winType]
```

Comment:
In future, please leave this over multiple lines to avoid diff noise when other mono-parametric windows are added..